dev/com.ibm.ws.messaging.common/resources/com/ibm/ws/sib/processor/CWSIPMessages.nlsprops

```diff
@@ -1847,7 +1847,7 @@ HEALTHSTATE_STATUS_CWSIP0908.useraction=CMSG0002
 # {0} Exception class causing the blockage
 # {1} Target destination
 HEALTHSTATE_STATUS_CWSIP0919=CWSIP0919W: Blocked receiver, the receiver cannot deliver the current inbound message for reason {0} on destination {1}
-HEALTHSTATE_STATUS_CWSIP0919.explanation=The reciever is unable to deliver a message to the target destination for the given reason, until this problem is resolved delivery of subsequent messages will be blocked.
+HEALTHSTATE_STATUS_CWSIP0919.explanation=The receiver is unable to deliver a message to the target destination for the given reason, until this problem is resolved delivery of subsequent messages will be blocked.
```

Comment:
suggested edit:
The receiver is unable to deliver a message to the target destination for the given reason. Until this problem is resolved, delivery of subsequent messages is blocked

Reply:
I'm ok making this change here, I'm not the owner of these messages so I don't want to do too much here other than fix the blatant spelling issue. This is just the explanation and not something that ends up getting returned to the user during runtime so seems reasonable for this change.